# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dvd+rwtools/glibc-2.9.patch
# Copyright (C) 2007 - 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---
--- ./growisofs.c.orig        2006-09-24 19:28:53.000000000 +0200
+++ ./growisofs.c    2008-01-08 00:56:57.000000000 +0100
@@ -418,6 +418,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <assert.h>
+#include <limits.h>
 #include "mp.h"
 
 #if defined(__unix) || defined(__unix__)
--- ./transport.hxx.orig      2006-09-23 13:29:07.000000000 +0200
+++ ./transport.hxx  2008-01-08 00:57:36.000000000 +0100
@@ -137,6 +137,7 @@
 #include <sys/wait.h>
 #include <sys/utsname.h>
 #include <scsi/sg.h>
+#include <limits.h>
 #if !defined(SG_FLAG_LUN_INHIBIT)
 # if defined(SG_FLAG_UNUSED_LUN_INHIBIT)
 #  define SG_FLAG_LUN_INHIBIT SG_FLAG_UNUSED_LUN_INHIBIT

